modules/encoders/ruby/md5.rb

```diff
@@ -0,0 +1,39 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Encoder
+  Rank = GreatRanking
+
+  def initialize
+    super(
+      'Name'             => 'Ruby Md5 Encoder',
```

Comment:
MD5 is a one-way hashing algorithm. I'm not sure that it would be useful as an encoder.

Reply:
You're right. So I made changes to md5. I set the md5 algorithm to base32.

Reply:
> I set the md5 algorithm to base32.

You what.